dist/ripple.js

```diff
@@ -1,51 +1,55 @@
-/*! Ripple.js v1.2.1
+/*! Ripple.js v1.3.0
  * The MIT License (MIT)
  * Copyright (c) 2014 Jacob Kelley */
 
-;(function($, document, Math){
-    $.ripple = function(selector, options) {
+;(function($, Math){
 
-        var self = this;
+    var selectors = [];
+    var defaults = {
+        debug: false,
+        on: 'mousedown',
 
-        var _log = self.log = function() {
-            if(self.defaults.debug && console && console.log) {
-                console.log.apply(console, arguments);
-            }
-        };
+        opacity: 0.4,
+        color: "auto",
+        multi: false,
+
+        duration: 0.7,
+        rate: function(pxPerSecond) {
+            return pxPerSecond;
+        },
+
+        easing: 'linear'
+    };
 
-        self.selector = selector;
-        self.defaults = {
-            debug: false,
-            on: 'mousedown',
+    $(function ($) {
+        $.ripple('[data-ripple-init]');
```

Comment:
But somebody need. It's cool to use plugin without any calls.
If any user do not need this, he will not use data-ripple-init attribute.